remove_item.md

```diff
@@ -1,54 +1,77 @@
 # Remover item
```

Comment:
Could you please put the number and icon of the topic?
# \#18: 🗑 Remove item

Tks!